README.md

```diff
@@ -511,6 +502,151 @@ in: <header>
 | x-exosite-prefix       | string | `token`       | apiKey   | A string to prefix in the header content.<br>E.g. X-secret: token `<user token>`                                        |
 | x-exosite-from         | string | `token`       | apiKey   | **Required**. The parameter name containing the token value to add in the Authentication header or query parameter. |
 
+##### Basic
+
+[Standard Basic Authentication](https://tools.ietf.org/html/rfc7617).
+
+Example:
+```yaml
+type: basic
+x-exosite-user-field: <account>
+x-exosite-secret-field: <secret>
+```
+
+##### APIKey
+
+Flexible API Key (Token) authentication:
+
+Example:
+```yaml
+type: apiKey
+name: secretHeader
+in: header
+x-exosite-prefix: Token
+x-exosite-from: secret
+```
+
+Will generate header:
+```
+secretHeader: Token <value of secret parameter>
+```
+
+##### Bearer
+
+For simplicity Murano added a `custom` authentication scheme following the [Bearer standard](https://tools.ietf.org/html/rfc6750), similar as Swagger V3.0 however missing in the version 2.0.
+
+Example:
+```yaml
+type: bearer
+name: <source parameter name>
+```
+
+Will generate header:
+```
+authorization: Bearer <value of secret parameter>
+```
+
+##### Signature
+
+Custom signature security signing the request in a header.
+
+Example:
+```yaml
+type: signature
+# name: <name of header, default signature>
+```
+
+Will generate header:
+```
+signature: Murano <Murano Key>:<signature>
+```
+
+See our [complete example](./examples/muranoauth.yaml).
+
+The `<signature>` is built as follow:
+```
+<signature> = Base64(RSA-SHA256(<host header><url (path with query parameter)><x-exosite-tracking-id header><date header><content-length header, if any>))
+```
+
+**Server example:**
+
+First you will need to get the [Murano public certificate](./murano-services-api-cert.pem) and in order to be called from Murano the service needs a service valid certificate.
```

Comment:
the `pem` file is still missing, which makes this link broken. other than this, this commit lgtm.

Reply:
@johnlinvc can you send me the public cert we will use for staging/production? thanks